### PR TITLE
feat(win&linux): implement the option to control gesture navigation

### LINKED ISSUE #895: Request to Disable Swipe Navigation by Default
**Is your feature request related to a problem? Please describe.**
I've noticed that the wry library enables swipe navigation by default, which can make the user interface unpredictable and difficult to control in certain situations.

**Describe the solution you'd like**
I would like to request that the wry library be updated to disable the ```SetIsSwipeNavigationEnabled``` option by default. This will ensure that the behavior of the application is always predictable and prevent unintended navigation or actions when the user is scrolling or swiping on the screen.

**Describe alternatives you've considered**
One alternative that could be considered is to provide an option in the library's settings to enable or disable swipe navigation, allowing developers to customize the behavior of the application as needed. However, I believe that disabling this feature by default is the safest and most predictable approach.

**Would you want to assign yourself to implement this feature?**
- [ ] Yes
- [x] No

**Additional context**
tauri-apps/tauri#5958

### PR DESCRIPTION
**Pull request by amrbashir**
### What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Docs
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### Checklist
- [X] This PR will resolve #895
- [x] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/wry/blob/dev/.changes/readme.md).
- [X] I have added a convincing reason for adding this feature, if necessary
- [X] It can be built on all targets and pass CI/CD.

### CODE REVIEW

**Comment by wusyong** [comment]
Should we bring this to other platforms as well? Or Windows-only fix could resolve that.

**Comment by amrbashir** [comment]
Should be fine now, I realized we already have this option for macOS so I removed the new APIs and implemented it for Windows and Linux.

File: src/webview/wkwebview/mod.rs [hunk 1]
```diff
@@ -313,9 +313,6 @@ impl InnerWebView {
 
       let _: id = msg_send![_preference, setValue:_yes forKey:NSString::new("allowsPictureInPictureMediaPlayback")];
 
-      #[cfg(target_os = "macos")]
-      let _: id = msg_send![_preference, setValue:_yes forKey:NSString::new("tabFocusesLinks")];
```
Comment (wusyong) [inline src/webview/wkwebview/mod.rs]: Why is this removed?

Comment (amrbashir) [reply src/webview/wkwebview/mod.rs]: my bad, removed by accident, should be good now

**Review by wusyong — APPROVED** [review]
